src/main/java/Messenger/MessengerNotification.java

```diff
@@ -0,0 +1,57 @@
+package Messenger;
+
+import Notification.Notification;
+
+import java.util.Date;
+
+
+/**
+ * Created by kenjik on 06.11.17.
+ */
+public class MessengerNotification implements Notification {
```

Comment:
Couldn't this be immutable? I would make fields `final` and remove setters since there won't be any use case for changing data in object after its initialization. As a rule of thumb I would choose making every class immutable if there aren't any strong reasons not to.

Reply:
Setters and getters are needed for Jackson serialization. Here - https://stackoverflow.com/questions/8367312/serializing-with-jackson-json-getting-no-serializer-found
If you know how to change the so called Visibility - I'm into it.

Reply:
No, Jackson doesn't require setters to do that, you can do that with Constructor and there also different methods. So you can remove setters